runtime/v2/runc/task/service.go

```diff
@@ -641,9 +641,6 @@ func (s *service) send(evt interface{}) {
 }
 
 func (s *service) handleProcessExit(e runcC.Exit, c *runc.Container, p process.Process) {
-	s.mu.Lock()
```

Comment:
I think we might need nolock version of `handleProcessExit`. And if the `p.Status` is stopped, we should fast-return to prevent from duplicated exit events.

Reply:
Thanks for your comment. I will fix it.

Reply:
I'm a little confused that in which cases there will be duplicated exit events?

Reply:
>  prevent from duplicated exit events.

Sorry for the unclear comments. There is no duplicate events.